.jshintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "eqeqeq": true
```

Comment:
Of course: I'm using it - instacool. And you can fine-tune rules and say what's a warning and what a real error and stuff like this. I think I'd favor it, because I'm using it, but in general jshint is fine :)